### ISSUE TITLE
jsdoc/empty-tags causes syntax errors for single-line JSDoc

### ISSUE THREAD

**Issue by maxpatiiuk** [gajus/eslint-plugin-jsdoc#1697]
## Expected behavior

jsdoc/empty-tags should not introduce syntax errors

## Actual behavior

If `@private` JSDoc tag is used in a single line JSDoc (`/** @private A */`) it gets replaced with ` @private`/, which causes a syntax error due to missing `/**` and `*/`)

## ESLint Config

```js
import jsdoc from 'eslint-plugin-jsdoc';

const config = [
  {
    files: ['**/*.js'],
    plugins: {
      jsdoc,
    },
    rules: {
      'jsdoc/empty-tags': 'error',
    },
  },
];

export default config;

```

## ESLint sample

```js
/** @private A */
const a = "a";
```

## Environment

Reproduced in stackblitz: https://stackblitz.com/edit/stackblitz-starters-suyi6vq3?file=README.md

- Node version: 22, 24
- ESLint version 10.4.1
- `eslint-plugin-jsdoc` version: 63
